src/System.Security.Cryptography.Csp/src/System/Security/Cryptography/CapiHelper.cs

```diff
@@ -799,11 +799,11 @@ internal static void DecryptKey(SafeKeyHandle safeKeyHandle, byte[] encryptedDat
             VerifyValidHandle(safeKeyHandle);
             if (null == encryptedData)
             {
-                throw new CryptographicException(SR.Format(SR.Argument_InvalidValue, "Encrypted Data is null"));
+                throw new CryptographicException(SR.Argument_InvalidValue + " Encrypted Data is null");
             }
             if (encryptedDataLength < 0)
             {
-                throw new CryptographicException(SR.Format(SR.Argument_InvalidValue, "Encrypted data length is less than 0"));
+                throw new CryptographicException(SR.Argument_InvalidValue + " Encrypted data length is less than 0");
```

Comment:
The extra literal string can happily be the message to Debug.Assert, though :smile: